content/vendor/guides/configuring-application-snapshots.md

```diff
@@ -0,0 +1,345 @@
+---
+date: "2020-09-06T00:00:00Z"
+lastmod: "2020-09-06T00:00:00Z"
+title: "Configure Application Snapshots"
+weight: "1005"
+---
+
+KOTS provides [snapshot capabilities](https://kots.io/vendor/snapshots/overview/) by leveraging the [Velero](https://velero.io/) open source project.
+
+## Objective 
+
+The purpose of this guide is to walk you through the nescessary steps to enable and configure snaphots in KOTS. 
+If you want to follow along, we'll be using the [AppDirect](https://github.com/replicatedhq/kotsapps/tree/master/app-direct) sample application.
+
+## Overview
+
+The guide is divided into three parts. The first part includes the steps performed by the software vendor, or similar organization that is packaging an application for delivery using Replicated KOTS.
+The second and third parts include steps that would be performed by the end user or customer.
+
+  [Part I - Enabling Snashots in KOTS](#part-i---enabling-snaphots-in-kots) Includes steps to enable snapshots in an application and configure the backup process.
+
+  [Part II - Application Configuration](#part-ii---application-configuration) This part covers the steps nescessary to enable and configure snaphots in the deployed application.
+ 
+  [Part III - Working with Snaphots](#part-iii---working-with-snaphots) This part covers tasks that are performed after the intial configuration.
+
+
+## Part I - Enabling Snaphots in KOTS
+
+
+These steps enable application snaphots by the vendor, which at a high level are:
+
+- [Add Backup Resource File](#add-backup-resource-file) - Adding this file pratcially enables snaphots in the application
+- [Configure Volumes](#configure-volumes) - Volumes are only part of the snapshot if they are annotated to be included.
+- [Configure Backup Hooks](#optionaladvanced-use-case-backup-hooks) - - May not be applicable to every situation, but these allow you to run commands prior to a backup.
+
+
+### Add Backup Resource File
+
+Create a new file named backup.yaml and add the following content:
+
+```shell
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: backup
+spec: {}
+
+```
+
+This is the minimal configuration and will suffice. 
+For further details on all of the available specs, please check the Velero documentation [here](https://velero.io/docs/v1.3.2/api-types/backup/)
+
+### Configure Volumes
+
+Volumes by default are not part of the snapshot.
+You configure which volumes to be part of the snaphot by adding the ```backup.velero.io/backup-volumes: ``` label annotation.
+
+In the AppDirect sample application, we have a PostgreSQL stateful set which has the following Volume:
```

Comment:
You reference this application, but there's no background or context (at least, not that I picked up on naturally trying to run this -- I usually end up skipping to the code samples). How about linking AppDirect to the kotsapp or some other doc that explains that it's a demo app?